google/cloud/telcoautomation/README.md

```diff
@@ -1,8 +1,8 @@
 # Telco Automation API C++ Client Library
 
 This directory contains an idiomatic C++ client library for the
-[Telco Automation API][cloud-service-docs], a service to APIs to automate 5G
-deployment and management of cloud infrastructure and network functions.
+[Telco Automation API][cloud-service-docs]. APIs to automate 5G deployment and
```

Comment:
s/. APIs/-- APIs/ ?

Reply:
PTAL.  I am almost convinced to remove this sentence altogether.

Reply:
> I am almost convinced to remove this sentence altogether.

+1. Let's get out of the documenting service descriptions business.